src/utils.test.js

```diff
@@ -573,6 +578,175 @@ describe('isFormEditing', () => {
   });
 });
 
+describe('getNextSelectedRowsState', () => {
+  it('should add a row to selectedRows if not selected', () => {
+    const selectedRows = {};
+    const row = { id: 1, name: 'John Doe' };
+
+    const result = getNextSelectedRowsState(selectedRows, row);
+
+    expect(result).toEqual({ [row.id]: row });
+  });
+
+  it('should remove a row from selectedRows if already selected', () => {
+    const row = { id: 1, name: 'John Doe' };
+    const selectedRows = { [row.id]: row };
+
+    const result = getNextSelectedRowsState(selectedRows, row);
+
+    expect(result).toEqual({});
+  });
+
+  it('should not mutate the original selectedRows object', () => {
+    const row = { id: 1, name: 'John Doe' };
+    const selectedRows = { [row.id]: row };
+
+    const result = getNextSelectedRowsState(selectedRows, row);
+
+    expect(selectedRows).toEqual({ [row.id]: row });
+    expect(result).not.toBe(selectedRows);
+  });
+});
+describe('isPrintable', () => {
+  it('returns true when pickSlips contain a match for requestId', () => {
```

Comment:
Could you name tests cases that starts from 'should' as we asked before for all tests?